examples/multigpu/graphbolt/node_classification.py

```diff
@@ -393,6 +404,10 @@ def parse_args():
     # Load and preprocess dataset.
     dataset = gb.BuiltinDataset("ogbn-products").load()
 
+    # Thread limiting to avoid resource competition.
```

Comment:
@peizhou001 should this done in this PR or somewhere in graphbolt library?

Reply:
Should be in this PR.